src/config_libconfig.c

```diff
@@ -384,8 +384,12 @@ char *parse_config_libconfig(options_t *opt, const char *config_file, bool *shad
 	lcfg_lookup_bool(&cfg, "shadow-ignore-shaped", &opt->shadow_ignore_shaped);
 	// --detect-rounded-corners
 	lcfg_lookup_bool(&cfg, "detect-rounded-corners", &opt->detect_rounded_corners);
-	// --xinerama-shadow-crop
-	lcfg_lookup_bool(&cfg, "xinerama-shadow-crop", &opt->xinerama_shadow_crop);
+	// --crop-shadow-to-monitor
+	lcfg_lookup_bool(&cfg, "crop-shadow-to-monitor", &opt->crop_shadow_to_monitor);
+	if (lcfg_lookup_bool(&cfg, "xinerama-shadow-crop", &opt->crop_shadow_to_monitor)) {
+		log_warn("xinerama-shadow-crop is deprecated. Use crop-shadow-to-monitor "
+		         "instead.");
+	}
```

Comment:
I would switch the order here so that the new `crop-shadow-to-monitor` has higher priority in the config file.

To be more in line with the other messages, I like `The xinerama-shadow-crop option has been deprecated. Please replace it with crop-shadow-to-monitor in your config file.` more. :wink:

Reply:
> I would switch the order here so that the new `crop-shadow-to-monitor` has higher priority in the config file.

addressed. i was thinking about the order but it turned out that i was wrong.